src/main/java/minicraft/level/tile/farming/CarrotTile.java

```diff
@@ -0,0 +1,32 @@
+package minicraft.level.tile.farming;
+
+import minicraft.gfx.Screen;
+import minicraft.gfx.SpriteLinker.LinkedSprite;
+import minicraft.gfx.SpriteLinker.SpriteType;
+import minicraft.level.Level;
+import minicraft.level.tile.Tiles;
+
+public class CarrotTile extends PlantTile {
+	private final LinkedSprite[] spritStages = new LinkedSprite[] {
+		new LinkedSprite(SpriteType.Tile, "carrot_stage0"),
+		new LinkedSprite(SpriteType.Tile, "carrot_stage1"),
+		new LinkedSprite(SpriteType.Tile, "carrot_stage2"),
+		new LinkedSprite(SpriteType.Tile, "carrot_stage3")
+	};
+
+    public CarrotTile(String name) {
+        super(name, null);
+    }
+
+    @Override
+    public void render(Screen screen, Level level, int x, int y) {
+        int age = (level.getData(x, y) >> 3) & maxStage;
+        Tiles.get("Farmland").render(screen, level, x, y);
+		int stage;
+		if (age < 2) stage = 0;
+		else if (age < 4) stage = 1;
+		else if (age < 7) stage = 2;
+		else stage = 3;
+		screen.render(x * 16, y * 16, spritStages[stage]);
+    }
+}
```

Comment:
The lack of unoriginality here is striking!

Reply:
Unoriginality?

Reply:
I have removed beetroot and added heavenly berries and hellish berries.